### PR TITLE
Fixed search and added new data models to search

### PR DESCRIPTION
**Pull request by clundin55**

### CODE REVIEW

File: yw_website/.gitignore [hunk 1]
```diff
@@ -15,6 +15,7 @@
 
 # Generated files
 .idea/**/contentModel.xml
+.idea/
```
Comment (aniehuser) [inline yw_website/.gitignore:18]: we might just want to ignore all .idea files: `.idea/**`

Comment (clundin55) [reply yw_website/.gitignore:18]: We have to ignore the folder as well, not just the contents

File: yw_website/yw_website/apps/website/models.py [hunk 2]
```diff
@@ -89,7 +89,6 @@ class Meta:
         unique_together = ("data_id", "run")
 
     data_id = models.IntegerField()
-    name = models.TextField()
```
Comment (aniehuser) [inline yw_website/yw_website/apps/website/models.py:92]: why is this being removed?

Comment (clundin55) [reply yw_website/yw_website/apps/website/models.py:92]: This field is a duplicate

Comment (aniehuser) [reply yw_website/yw_website/apps/website/models.py:92]: it does not look like a duplicate to me

Comment (aniehuser) [reply yw_website/yw_website/apps/website/models.py:92]: ope nevermind its below `in_program_block`

**Review by aniehuser — APPROVED** [review]

**Pull request merged**